app/components/forms/session-speaker-form.js

```diff
@@ -357,22 +357,6 @@ export default Component.extend(FormMixin, {
             {
               type   : 'empty',
               prompt : this.get('l10n').t('Please enter url of website')
-            },
-            {
-              type   : 'regExp',
-              value  : compulsoryProtocolValidUrlPattern,
-              prompt : this.get('l10n').t('Please enter a valid url')
-            }
-          ]
-        },
-        website: {
-          identifier : 'speaker_website',
-          optional   : true,
-          rules      : [
-            {
-              type   : 'regExp',
-              value  : compulsoryProtocolValidUrlPattern,
-              prompt : this.get('l10n').t('Please enter a valid url')
```

Comment:
@Anupam-dagar Instead of removing the validations completely we can take example from this file https://github.com/fossasia/open-event-frontend/blob/c5b864a63919464fe7cf0ed36d5a946bdcede02d/app/components/forms/admin/content/social-links-form.js#L13-L36 and make our validation rules according to what we do what admin/content/social-links.